modules/ROOT/pages/storage.adoc

```diff
@@ -22,10 +24,11 @@ storage:
     # device.
     wipe_table: false
     partitions:
+    - number: 4
+      # Allocate at least 5G to the rootfs. See NOTE above about this.
```

Comment:
This and the f-c-c PR are implicitly committing to a 5 GiB minimum.  Should we socialize that a bit more?  Are we confident it'll be large enough in the long term?

Reply:
Let's keep discussing this in https://github.com/coreos/fedora-coreos-tracker/issues/586.

Reply:
Updated to use 8 GiB now as per https://github.com/coreos/fedora-coreos-tracker/issues/586#issuecomment-785254764.